components/brave_wallet/browser/swap_response_parser.cc

```diff
@@ -660,6 +657,12 @@ mojom::LiFiQuotePtr ParseQuoteResponse(const base::Value& json_value) {
       route->steps.push_back(std::move(step));
     }
 
+    std::vector<std::string> tools;
+    for (const auto& step : route->steps) {
+      tools.push_back(step->tool);
+    }
+    route->unique_id = base::JoinString(tools, "-");
+
```

Comment:
Is there a test coverage for this?

Reply:
The existing tests have been modified to test for `unique_id` as well.